src/Administration/Resources/app/administration/src/app/component/form/select/base/sw-select-base/sw-select-base.scss

```
@@ -109,23 +117,22 @@ $sw-select-focus-transition: all ease-in-out 0.2s;
 
     &.is--disabled {
         .sw-block-field__block {
-            background-color: $color-gray-100;
+            background-color: var(--color-background-primary-disabled);
         }
 
         .sw-label {
-            background-color: $color-gray-100;
+            background-color: var(--color-text-primary-disabled);
```

Should this be:
```suggestion
            color: var(--color-text-primary-disabled);
```

Not sure where this styles apply tbh 😄

---

I corrected that. The `sw-label` component are these boxes/values inside the multi select.